# HG changeset patch
# User Rik <rik@octave.org>
# Date 1430497932 25200
#      Fri May 01 09:32:12 2015 -0700
# Node ID 0afc06948ba724c4ab6dfa902397cdc7080c0568
# Parent  f5e65f806123701ecdd0bd700bf1fcfa1300bee5
acinclude.m4: Use simpler AC_PREPROC_IFELSE test for checking Qscintilla version.

* acinclude.m4 (OCTAVE_CHECK_VERSION_2_6_0): Replace AC_COMPILE_IFELSE with
AC_PREPROC_IFELSE for Qscintilla version check.

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -286,17 +286,17 @@ dnl FIXME: This test uses a version numb
 dnl        be re-written to actually call the function, but is it worth it?
 dnl
 AC_DEFUN([OCTAVE_CHECK_VERSION_2_6_0], [
   AC_CACHE_CHECK([whether Qscintilla has version 2.6.0 or later],
     [octave_cv_version_2_6_0],
     [AC_LANG_PUSH(C++)
     ac_octave_save_CPPFLAGS="$CPPFLAGS"
     CPPFLAGS="$QT_CPPFLAGS $CPPFLAGS"
-    AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
+    AC_PREPROC_IFELSE([AC_LANG_PROGRAM([[
         #include <Qsci/qsciglobal.h>
         ]], [[
         #if QSCINTILLA_VERSION < 0x020600
         #error Old FindFirst function found.
         #endif
         ]])],
       octave_cv_version_2_6_0=yes,
       octave_cv_version_2_6_0=no)
